src/cpp/session/modules/SessionDataViewer.R

```diff
@@ -890,3 +901,8 @@
    invisible("")
 })
 
+.rs.addFunction("isZeroLength", function(x)
```

Comment:
This doesn't feel quite right to me, since all R objects have a length and e.g.

```
> length(NA)
[1] 1
```

Could we instead reverse the check, and just confirm something like:

```
.rs.addFunction("isNonemptyScalarString", function(x)
{
  is.character(x) && length(x) == 1 && nzchar(x)
})
```

since that's more-or-less what `get()` and `assign()` require in their usages?

Reply:
Thanks for this suggestion! My lack of R knowledge is showing 😅 

TIL all objects have a length. I think this is going to be a helpful fact for me to carry forward!

Makes more sense to check for non-empty string and this covers our bases if something like `NA` or `NULL` is passed.